Volume/Views/Orgs Admin/OrgFlyerCellView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  OrgFlyerCellView.swift
+//  Volume
+//
+//  Created by Cindy Liang on 11/4/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct OrgFlyerCellView: View {
+    // MARK: - Properties
+
+    let flyer: Flyer
+    let navigationSource: NavigationSource
+
+    @StateObject var urlImageModel: URLImageModel
+    @EnvironmentObject private var userData: UserData
+    @ObservedObject var viewModel: FlyersView.ViewModel
+    
+    // MARK: - Constants
+
+    private struct Constants {
+        static let horizontalSpacing: CGFloat = 8
+        static let verticalSpacing: CGFloat = 8
+    }
+    
+    var body: some View {
+        
+        HStack(alignment: .top, spacing: Constants.horizontalSpacing) {
+            imageFrame
+
+            VStack(alignment: .leading, spacing: 8) {
+                organizationName
+                flyerTitle
+                flyerDate
+                flyerLocation
+            }
+        }
+        .padding(.bottom, 16)
+        
+    }
+    private var imageFrame: some View {
+        ZStack(alignment: .center) {
+            if let flyerImage = urlImageModel.image {
+                Color(uiColor: flyerImage.averageColor ?? .gray)
+
+                Image(uiImage: urlImageModel.image ?? UIImage())
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+            } else {
+                SkeletonView()
+                    .shimmer(.mediumShimmer())
+            }
+        }
+        .frame(width: 80, height: 80)
+    }
+    private var organizationName: some View {
+        HStack(alignment: .top) {
+            Text(flyer.organization.name)
+                .font(.newYorkMedium(size: 10))
+                .lineLimit(2)
+
+            Spacer()
+            HStack(alignment: .center, spacing: 2) {
+                Circle()
+                    .fill().foregroundColor(Color.gray)
+                    .frame(width: 3, height: 3)
```

Comment:
May or may or not be good to factor this number out into a constant

Reply:
@cindy-x-liang I would filter this out and call it `circleSize: CGFloat = 3`